QuadradiusR/scripts/full_power_list.gd

```diff
@@ -0,0 +1,46 @@
+extends PanelContainer
```

Comment:
messy code begin: if i remember correctly, this one is for all the powers owned by *player*, not individual toruses. We will not use this in this task, as it was moved to new issue #272 (if i remember right).